docs/two-phase-commit-transactions.md

```diff
@@ -44,82 +38,83 @@ scalar.db.username=cassandra
 
 ### Get a `TwoPhaseCommitTransactionManager` instance
 
-First, you need to get a `TwoPhaseCommitTransactionManager` instance to execute Two-phase Commit Transactions.
+You first need to get a `TwoPhaseCommitTransactionManager` instance to execute two-phase commit transactions.
 
-You can use `TransactionFactory` to get a `TwoPhaseCommitTransactionManager` instance as follows:
+To get a `TwoPhaseCommitTransactionManager` instance, you can use `TransactionFactory` as follows:
 
 ```java
-TransactionFactory factory = TransactionFactory.create("<configuration file path>");
+TransactionFactory factory = TransactionFactory.create("<CONFIGURATION_FILE_PATH>");
 TwoPhaseCommitTransactionManager transactionManager = factory.getTwoPhaseCommitTransactionManager();
 ```
 
-### Begin/Start a transaction (for coordinator)
+### Begin or start a transaction (for Coordinator)
 
-You can begin/start a transaction as follows:
+For the process or application that begins the transaction to act as Coordinator, you should use the following `begin` method:
 
 ```java
-// Begin a transaction
+// Begin a transaction.
 TwoPhaseCommitTransaction tx = transactionManager.begin();
+```
 
-Or
+Or, for the process or application that begins the transaction to act as Coordinator, you should use the following `start` method:
 
-// Start a transaction
+```java
+// Start a transaction.
 TwoPhaseCommitTransaction tx = transactionManager.start();
 ```
 
-The process/application that begins the transaction acts as a coordinator, as mentioned.
-
-You can also begin/start a transaction by specifying a transaction ID as follows:
+Alternatively, you can use the `begin` method for a transaction by specifying a transaction ID as follows:
 
 ```java
-// Begin a transaction with specifying a transaction ID
-TwoPhaseCommitTransaction tx = transactionManager.begin("<transaction ID>");
+// Begin a transaction by specifying a transaction ID.
+TwoPhaseCommitTransaction tx = transactionManager.begin("<TRANSACTION_ID>");
+```
 
-Or
+Or, you can use the `start` method for a transaction by specifying a transaction ID as follows:
 
-// Start a transaction with specifying a transaction ID
-TwoPhaseCommitTransaction tx = transactionManager.start("<transaction ID>");
+```java
+// Start a transaction by specifying a transaction ID.
+TwoPhaseCommitTransaction tx = transactionManager.start("<TRANSACTION_ID>");
 ```
 
-Note that you must guarantee uniqueness of the transaction ID in this case.
+{% capture notice--info %}
+**Note**
+
+You must guarantee the uniqueness of the transaction ID in this case. To get the transaction ID with `getId()`, you can specify the following:
 
-And, you can get the transaction ID with `getId()` as follows:
 ```java
 tx.getId();
 ```
+{% endcapture %}
 
-### Join the transaction (for participants)
+<div class="notice--info">{{ notice--info | markdownify }}</div>
```

Comment:
@brfrn169 Similar to the changes we made in `api-guide.md` in https://github.com/scalar-labs/scalardb/pull/1084/, should we move this note to the `### Join a transaction (for participants)` section and add it to the `### Resume a transaction` section?

Ref: 
- https://github.com/scalar-labs/scalardb/blob/e577fdd1cf63df00646a0a41bb6e0b479728ba8d/docs/api-guide.md?plain=1#L372
- https://github.com/scalar-labs/scalardb/blob/e577fdd1cf63df00646a0a41bb6e0b479728ba8d/docs/api-guide.md?plain=1#L395

Reply:
Yes please. Thanks!